--- gmime-3.2.13/configure.ac	2022-08-18 08:48:43.000000000 -0500
+++ gmime-3.2.13/configure.ac	2023-08-10 14:14:22.152450428 -0500
@@ -599,14 +599,9 @@
 build/vs2010/Makefile
 build/vs2010/config-win32.h
 build/vs2017/Makefile
-docs/Makefile
-docs/reference/Makefile
-docs/tutorial/Makefile
-examples/Makefile
 util/Makefile
 gmime/Makefile
 gmime/gmime-version.h
-tests/Makefile
 tools/Makefile
 gmime.spec
 gmime.pc
